template/ApiException.mustache

```diff
@@ -30,45 +31,20 @@ use \Exception;
  */
 class ApiException extends Exception
 {
+    protected ResponseInterface $responseObject;
 
-    /**
-     * The deserialized response object
-     *
-     * @var $responseObject;
-     */
-    protected $responseObject;
-
-    /**
-     * Constructor
-     *
-     * @param string        $message         Error message
-     * @param int           $code            HTTP status code
-     * @param string[]|null $responseHeaders HTTP response header
```

Comment:
Why were there 2 more arguments in the constructor?

Reply:
Because we refactored these fields with Przemo, and ApiException was accepting these fields. I removed phpdoc because types are enough in constructor.